src/components/SocialCard.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react"
+import SocialCardDisplay from "./SocialCardDisplay";
+import {
+    TiSocialFacebookCircular,
+    TiSocialGithub,
+    TiSocialTwitterCircular,
+    TiSocialLinkedin,
+} from "react-icons/ti";
+
+const SocialCardOptions = [
+    {
+        name : "Facebook",
+        logo : <TiSocialFacebookCircular className="float-right" size="3rem" />,
```

Comment:
Divide the card into section for heading, subheading and icon, followed by description. Don't use float.